intel-retail/rtsf-at-checkout-reference-design
# Rest volume

## PR Checklist



## What are you changing?

device-rest and device-mqtt service weren't loading resources due to missing volumes

## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

1. make docker
2. make run-base
3. open postman and call "http://localhost:59881/api/v3/device/all"
4. see all devices listed in the .yaml file are present

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/rtsf-at-checkout-reference-design )
